web/src/app/shared/models/activity.model.ts

```diff
@@ -1,4 +1,4 @@
-import { RepositoryModel } from './repository.model';
+import { RepositoryModel } from './index.model';
```

Comment:
Same here, original import is fine as its loading files in the same type/directory (in this case `model`s)

Reply:
done